### PR TITLE
07 - Make "Member Search" form

### PR DESCRIPTION
**Pull request by Baavi**
It connects to the DB, and works, but needs to finalize the DB layout before writing the query and is dependent on the error msg components to display the validation errors

### CODE REVIEW

File: editMember.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<body>
+    <?php
+    include_once("navbar.inc");
+    ?>
+    <div class="editmember">
+        <h1 class="white-text">Search for member </h1>
+        <form method="post" action="searchMEMBER.php" novalidate="novalidate">
+            <div class="inputField">
+                <!-- <label for="member_id">Search for member </label>-->
```
Comment (RoryHowlettUni) [inline editMember.php]: Was this for comparing the heading as a label vs a h1?? I'm not sure what the process is for doing design changes, might be a Khang question.

File: styles/addorder.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+#row_2 {
+  display: none;
+}
+.addorder,
+.editmember {
+  padding-top: 4rem;
+  margin: auto;
+  width: fit-content;
+}
```
Comment (RoryHowlettUni) [inline styles/addorder.css:83]: Is the Member Search form relying on the addorder.css file?? Should it have it's own CSS file or is this fine?

Comment (RoryHowlettUni) [reply styles/addorder.css:83]: The styling also doesn't currently match Figma either.

**Review by RoryHowlettUni — COMMENTED** [review]
The deletion of the global.css and styles.css in this branch may delete the respective files in main when this PR is merged, I say may because I'm not 100% if this is the case, it'd be worth looking into!

File: styles/addorder.css (hunk 2 above)
Comment (Baavi) [reply styles/addorder.css:83]: Yes, basically both pages need the same CSS code

**Pull request closed without merging**